hexod

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+docker run \
+  --interactive --tty --rm \
+  --volume "$PWD":/hexosite \
+  --workdir /hexosite \
+  -p 4000:4000 \
+  bestony/hexojs:latest "$@"
```

Comment:
Is it possible to wrap `hexo` inside so that users can run `hexod [command] [--args]` directly?

Reply:
Of course yes, I will push a new commit  again.